app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt

```diff
@@ -43,15 +44,18 @@ class SettingPresenter(
 
     private val versionNumber = BuildConfig.VERSION_NAME
 
-    private val autoLockObserver: Consumer<Boolean>
+    val autoLockObserver: Consumer<Boolean>
```

Comment:
I'm confused why this is a consumer, and everything else is an `Observer`, or a closure used to subscribe.

A tiny comment here would be helpful. 

```
// we use a Consumer here because…
```

Reply:
For each toggle button a Consumer is used as a callback in the subscribe method. This is because when switching the toggle button, there is no need for the onError or other events except onNext . As far as I understand, when only one event is needed, there is no reason to use an Observer and implement all the events (onNext, onError, onComplete..). Instead a Consumer can be used. I will add a comment if you think it's necessary.

Reply:
IMO the comment might not be necessary; we only use `Consumer`s in the `SettingPresenter`. It also has the advantage of tying the lifecycle to the view.

Reply:
:+1: